distros/fedora/openalpr.spec

```diff
@@ -0,0 +1,125 @@
+Name:     openalpr
+Version:  2.3.0
+Release:  1%{?dist}
+Summary:  OpenALPR
```

Comment:
A little bit more details would help. The summary should be a brief, one-line summary of the package.